src/DependencyInjection/Compiler/FormFactoryCompilerPass.php

```diff
@@ -14,6 +14,9 @@
 use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\FormPass;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
+/**
+ * @deprecated disable this compiler pass
```

Comment:
`deprecated since...` please follow the format described on the contributing guide.